src/gui/dialogs/export_project.ui

```diff
@@ -81,25 +81,57 @@
          </widget>
         </item>
         <item>
+         <widget class="QWidget" name="depthWidget" native="true">
+          <property name="minimumSize">
+           <size>
+            <width>0</width>
+            <height>20</height>
+           </size>
+          </property>
+          <layout class="QVBoxLayout">
+           <property name="margin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QLabel" name="labelBitDepth">
+             <property name="text">
+              <string>Depth:</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QComboBox" name="depthCB">
+             <item>
+              <property name="text">
+               <string>16 Bit Integer</string>
+              </property>
+             </item>
+             <item>
+              <property name="text">
+               <string>24 Bit Float</string>
```

Comment:
You're right. In the end all three methods use PCM to encode the signal. I only think that the term "Integer" already goes too much into the technical direction of programming. Put differently: a [search for "16 bit integer"](https://www.google.de/#q=16+bit+integer) will not be as informative for the end user as a [search "16 bit PCM"](https://www.google.de/#q=16+bit+pcm). The latter returns much more audio related results.